### ISSUE TITLE
Remove item link on Cart block has no hover state

### ISSUE THREAD

**Issue by nielslange** [woocommerce/woocommerce-gutenberg-products-block#10639]
## Describe the bug

The remove item link on the Cart block has no hover state at the moment.

## To reproduce

1. Create a test page and add the Cart block to it.
2. Go to the frontend and add a product to the cart.
3. Go to the test page with the Cart block.
4. Hover over the "Remove item" link. 
5. See that the link has no hover state.

> **Note**
> The underlying HTML code of the link is `<button>`. This might explain the missing hover state.

## Expected behavior

I expect the "Remove item" link to have a hover state.

**Comment by SameerJS6** [comment]
Could you please help me with the location of the file that needs changes? I am new here.
